### ISSUE TITLE
Bad UTF-8

### ISSUE THREAD

**Issue by XEGARE** [bruxo00/SAMPMailJS#1]
Hello! The Russian text in UTF-8 is not well coded.
It says - Information about you, of course in Russian)

https://i.imgur.com/lK8UCYj.png

**Comment by diogomartino** [comment]
I think it is a problem on the HTTP function on SAMP, I don't really think there is a fix for that unfortunately.

**Comment by KDOPE** [comment]
There is one. I'm from lithuania, and I have to send emails with lithuanian symbols, so I had to create custom function, which replaces lithuanian symbols with url encoding, for example: ą in url encoding stands for %C4%85. So I had to replace all these symbols before sending an http request. Alright. Http request had been sent, and now we should move to sampmail.js. I used javasript function to decode that url and change these (%C4%85) encoded symbols to normal ones, and now text again contains lithuanian symbols, like ą, č, etc. I recommend to use https://amp.urlencoder.org site for changing russian symbols to the url matching ones. Then put them into array, and make a loop, change all of russian symbols to these url codes. I will put my script later, when I will be at home.
